app/src/main/res/values/strings.xml

```diff
@@ -82,15 +82,15 @@
     <string name="d_checkCredentials_errorMessage_noAccessWithMessage">No access to API: %s</string>
     <string name="d_checkCredentials_errorMessage_unknown">Unknown error</string>
     <string name="d_checkCredentials_errorMessage_unknownWithMessage">Unknown error: %s</string>
-    <string name="wrongUsernameOrPassword_errorMessage">Couldn\'t log in: Presumably wrong username or password</string>
+    <string name="wrongUsernameOrPassword_errorMessage">Couldn\'t log in. Maybe the username or password is wrong?</string>
     <string name="previousUpdateNotFinished">Previous update not finished yet</string>
     <string name="feedName_favorites">Favorites</string>
     <string name="feedName_archived">Archived</string>
     <string name="feedName_unread">Unread</string>
     <string name="articleList_favoriteMark">Favorite mark</string>
     <string name="articleList_ArchivedMark">Archived mark</string>
     <string name="noPreviousArticle">There is no previous article in current list</string>
-    <string name="scrolledOverBottom">Scroll %d time(s) more over the bottom to mark the article as read</string>
+    <string name="scrolledOverBottom">Scroll %d more time(s) past the bottom to mark article as read</string>
```

Comment:
Could be pluralized.

Reply:
LGTM, what exactly do you mean here?

Reply:
[Plurals](https://developer.android.com/guide/topics/resources/string-resource#Plurals). I don't think that feature is used often enough to bother.